### PR TITLE
Update for LM2015 transition model

### PR DESCRIPTION
**Pull request by rois1995**
Hi everyone,

This work updates the transition portion with the LM2015 model and the SA-LM coupling following PR #1810. 

I am adding information on the Physical Definitions regarding the LM_OPTIONS entries.

I'll also add validation regarding SA-LM coupling.

### CODE REVIEW

File: _docs_v7/Physical-Definition.md [hunk 1]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
   - Curvature corrections are currently not implemented.
 
 Modifications from each of these three groups can be combined, for example `SST_OPTIONS= V2003m, VORTICITY, SUSTAINING`
+
+## Transition Models ##
+
+| Solver | Version |
+| --- | --- |
+| `*_RANS` | 7.4.0 |
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:166]:
```suggestion
| `*_RANS` | 7.5.0 |
```

File: _docs_v7/Physical-Definition.md [hunk 2]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+
+This section describes how to setup transition models for RANS simulations. Transition is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`.
+A transition model can then be selected via the option `KIND_TRANS_MODEL`.
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:169]:
```suggestion
This section describes how to setup transition models for RANS simulations. Transition is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS` together with a choice of `KIND_TRANS_MODEL` (different from `NONE`).
```

File: _docs_v7/Physical-Definition.md [hunk 3]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+Different submodels and correlations are specified via the different options listed below.
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:170]:
```suggestion
Currently, the only valid option for `KIND_TRANS_MODEL` is `LM`, for Langtry-Menter transition models.
Different submodels and correlations are then specified via `LM_OPTIONS` (for example `LM_OPTIONS= LM2015, MENTER_LANGTRY`).
```

File: _docs_v7/Physical-Definition.md [hunk 4]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+
+The following modifications are allowed:
+- Versions:
+  - `LM2015` - Correction to include stationary crossflow instabilities. It has to be used only in 3D problems. The RMS of roughness used in this model has to be set through the separate option `HROUGHNESS`.
+- Correlations:
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:175]:
```suggestion
- Correlations (only one can be specified):
```

File: _docs_v7/Physical-Definition.md [hunk 5]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+  - `MALAN` - This is the default correlation when the LM model is coupled with the `SA` turbulence model.
+  - `SULUKSNA` - This should be used only if the `SST` model is used. It requires a formulation of
+  - `KRAUSE` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear and inconsistent with literature.
+  - `KRAUSE_HYPER` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear.
+  - `MEDIDA` - Designed for `SA` turbulence model. Has problems when dealing with separation induced transition.
+  - `MEDIDA_BAEDER` - Designed for `SA` turbulence model. Has problems when dealing with separation induced transition.
+  - `MENTER_LANGTRY` - This is the default correlation when the LM model is coupled with the `SST` turbulence model.
+
+  Modifications from each of these three groups can be combined, for example `LM_OPTIONS= LM2015, MENTER_LANGTRY`. Although, at most one correlation should be chosen.
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:184]:
```suggestion
```

File: _docs_v7/Physical-Definition.md [hunk 6]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+This section describes how to setup transition models for RANS simulations. Transition is activated using the option `KIND_SOLVER= RANS`, or `KIND_SOLVER= INC_RANS`.
+A transition model can then be selected via the option `KIND_TRANS_MODEL`.
+Different submodels and correlations are specified via the different options listed below.
+
+The following modifications are allowed:
+- Versions:
+  - `LM2015` - Correction to include stationary crossflow instabilities. It has to be used only in 3D problems. The RMS of roughness used in this model has to be set through the separate option `HROUGHNESS`.
+- Correlations:
+  - `MALAN` - This is the default correlation when the LM model is coupled with the `SA` turbulence model.
+  - `SULUKSNA` - This should be used only if the `SST` model is used. It requires a formulation of
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:177]: incomplete sentence

File: _docs_v7/Physical-Definition.md [hunk 7]
```diff
@@ -158,3 +158,27 @@ The following modifications are allowed:
...
+  - `KRAUSE` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear and inconsistent with literature.
+  - `KRAUSE_HYPER` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear.
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:179]: Due to ambiguous literature?

Comment (rois1995) [reply _docs_v7/Physical-Definition.md:179]: Yes. The paper from which I took the KRAUSE correlation pointed to another paper where the correlation presented was different. Then, @sun5k  told me that the true correlation is yet another one and that he has access to it. Thus I hope he will modify it.

File: _docs_v7/Physical-Definition.md [hunk 8]
```diff
@@ -158,3 +158,25 @@ The following modifications are allowed:
...
+Different submodels and correlations are then specified via `LM_OPTIONS` (for example `LM_OPTIONS= LM2015, MENTER_LANGTRY`).
+
+The following modifications are allowed:
+- Versions:
+  - `LM2015` - Correction to include stationary crossflow instabilities. It has to be used only in 3D problems. The RMS of roughness used in this model has to be set through the separate option `HROUGHNESS`.
+- Correlations (only one can be specified):
+  - `MALAN` - This is the default correlation when the LM model is coupled with the `SA` turbulence model.
+  - `SULUKSNA` - This should be used only if the `SST` model is used. It requires a formulation of
+  - `KRAUSE` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear and inconsistent with literature.
+  - `KRAUSE_HYPER` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear.
```
Comment (pcarruscag) [inline _docs_v7/Physical-Definition.md:179]:
```suggestion
  - `KRAUSE` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear due to inconsistencies in the literature.
  - `KRAUSE_HYPER` - This correlation should be used for hypersonic flows. Its implementation at the moment is unclear due to inconsistencies in the literature.
```

**Review by pcarruscag — APPROVED** [review]

File: _docs_v7/Physical-Definition.md (hunk 6 above)
Comment (rois1995) [reply _docs_v7/Physical-Definition.md]: I also added some figures for the validation of the E387 test case. Please, check if those are good, and then I'll merge this, thank you.

**Review by pcarruscag — COMMENTED** [review]
Yep looks good

**Pull request merged**